docs/resources/connector_gcp_vpc.md

```diff
@@ -10,8 +10,16 @@ description: |-
 
 Manage GCP Cloud Connector.
 
+## Routing Options
```

Comment:
You don't edit this file directly. It will be generated by `make doc` from templates and source codes... You could add a new template in `templates/resources/` as `connector_gcp_vpc.md.tmpl` similar to others in the directory.. and add those descriptions in the template.